client/components/Orders.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom';
+import { fetchOrder } from '../store'
+
+class Orders extends Component {
+  constructor(props) {
+    super(props);
+  }
+  
+  async componentDidMount () {
+    try {
+      await this.props.fetchOrder()
```

Comment:
you forgot to pass in your userId!
```suggestion
await this.props.fetchOrder(this.props.auth.id)
```